benchmarks/experimental_components/examples/op_benchmark.py

```diff
@@ -0,0 +1,98 @@
+"""Example use of Timer and op fuzzers to measure kernel performance.
+
+$ python -m examples.op_benchmark
+"""
+
+import numpy as np
+import torch
+
+from utils import Timer
+from op_fuzzers.binary import BinaryOpFuzzer
+from op_fuzzers.unary import UnaryOpFuzzer
+
+
+_MEASURE_TIME = 1.0
+
+
+def assert_dicts_equal(dict_0, dict_1):
```

Comment:
is there a reason why built-in dict comparison won't work?

Reply:
It fails if the dict contains numpy arrays. (I added a docstring to that effect.)